lib/help.js

```diff
@@ -1,67 +1,149 @@
-const path = require('path');
-const fs = require('fs');
 const config = require('config');
+const Table = require('table-layout');
+const usage = require('command-line-usage');
+const commands = require('./commands');
 
-function pad(text) {
-  return text.padEnd(15);
+function styleCommand(command) {
+  return `${'$'.grey} ${config.cli} ${command.name}`;
 }
 
-exports.run = function() {
-  console.log('');
-  console.log(`Usage: ${config.cli} <command> [arguments]`);
-  const files = fs
-    .readdirSync(__dirname)
-    .map(file => {
-      const stats = fs.statSync(path.join(__dirname, file));
-      if (stats.isDirectory()) {
-        return fs.readdirSync(path.join(__dirname, file)).map(f => path.join(file, f));
+const owlbert = () => {
+  // http://asciiart.club
+  return `          ░                                        ░
+          ░░                                       ░░
+        ░░░░░                                     ░░░░ ░
+       ░░░░░░░                                    ░░░░░░
+       ▐░░░░░░▄                                 ░░░░░░░░░
+       ░░░░░░░░░▐░░░                        ░░░░░░░░░░░░░   ${config.cli.blue.bold}
+        ░░░░░░░░░░░░░░░░░▐▐▐░░░░░░░░░░▐▐░░░░░░░░░░░░░░░░░░
+      ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    ${
+        `a utlity for interacting with ReadMe`.bold
```

Comment:
This is a _ridiculously_ minor detail so feel free to ignore/punch me, but... we should adjust these lines so the text is aligned flush left. (Literally only off by a single space, I know, but my OCD just cannot abide it!)

Reply:
@erunion well that was extremely quick...

Reply:
gotta go fast 🏃💨